### PR TITLE
CI: Create static checks script

### LINKED ISSUE #6: Create central static checks test
Add a script that will be the **single** source of all static tests used to build components.

Initially, it will simply run [checkcommits](https://github.com/clearcontainers/tests/tree/master/cmd/checkcommits), but will be extended later to run linters, etc.

All other repos should invoke this script to avoid a proliferation of (different) static check scripts.

### PR DESCRIPTION
**Pull request by jodh-intel**
Add a script that will be the **single** source of all static tests
run before building components.

Initially, it simply runs
[checkcommits](https://github.com/clearcontainers/tests/tree/master/cmd/checkcommits),
but will be extended later to run linters, etc.

All other repos should invoke this script to avoid a proliferation of
(different) static check scripts.

Fixes #6.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: .ci/static-checks.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Description: Central script to run all static checks.
+#   This script should be called by all other repositories to ensure 
+#   there is only a single source of all static checks.
+
+set -e
+
+test_repo="github.com/clearcontainers/tests"
+
+# Clone Tests repository.
+go get "$test_repo"
```
Comment (grahamwhaley) [inline .ci/static-checks.sh]: don't know if you need any magic options on the `go get` here, maybe `-u` or `-d` and some judicious `...`'s ? - just asking, as I know we've hit such 'non-buildable repos issue warnings' situations before.

**Comment by jodh-intel** [comment]
Ping @sameo, @gnawux, @chavafg.

**Comment by jodh-intel** [comment]
ping @kata-containers/tests

File: .ci/static-checks.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Description: Central script to run all static checks.
+#   This script should be called by all other repositories to ensure 
+#   there is only a single source of all static checks.
+
+set -e
+
+test_repo="github.com/clearcontainers/tests"
```
Comment (WeiZhang555) [inline .ci/static-checks.sh]: Why not copy source codes here? I see they are only two golang source files.

**Comment by jodh-intel** [comment]
@WeiZhang555 - I agree - branch updated.

Note that I've also changed `.travis.yml` to test with golang 1.8 **and** the latest "stable" version.

**Comment by jodh-intel** [comment]
gah - looks like I need to sort out the deps for the repo...

**Comment by jodh-intel** [comment]
Fixed.

**Comment by jodh-intel** [comment]
Hi @grahamwhaley - could you take another look?

**Comment by WeiZhang555** [comment]
LGTM

**Pull request merged**